packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -276,7 +276,7 @@ export const [
     Headline1Weight,
     Headline1Size,
     Headline1LineHeight,
-    'serif'
+    'Noto Serif-fallback'
```

Comment:
Can you make constants for these?

Eventually they should be pulled from tokens instead as imports. The names should align as if they were tokens, ie `Headline1FallbackFontFamily`